GoogleForCreators/web-stories-wp
# Redesign: Page/background element selection

## Context



## Summary



## Relevant Technical Choices



## To-do



## User-facing changes
Background/page selection has been updated based on [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=3616%3A31887)


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add a background element.
2. Select it
3. Verify the selection looks as in [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=3616%3A31887)

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6203
